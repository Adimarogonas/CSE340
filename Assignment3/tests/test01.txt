x, y: int;
{
    x = < y 2;
}

