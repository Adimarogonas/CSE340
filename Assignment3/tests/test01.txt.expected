TYPE MISMATCH 3 C1
